update/Pos/Update/Worker.hs

```diff
@@ -12,6 +12,9 @@ import           Formatting (build, sformat, (%))
 import           Serokell.Util.Text (listJsonIndent)
 import           System.Wlog (logDebug, logInfo)
 
+import           Control.Lens (views)
```

Comment:
This import should be in the group above, it's not special in any sense, so it shouldn't be separated.